packages/era/src/helpers.ts

```diff
@@ -55,6 +56,26 @@ export const deserializeE2Store = (bytes: Uint8Array): e2StoreEntry[] => {
   return entries
 }
 
+
+function littleEndianToBigInt(bytes: Uint8Array) {
```

Comment:
does the ethjs `bytesToBigInt` function not work here?  It has a `littleEndian: boolean = false` parameter

Reply:
When I switch to `bytesToBigInt`, my tests start failing so something's awry.  Will research a bit more.